protostar/commands/test/test_execution_environment.py

```diff
@@ -88,6 +92,23 @@ def deploy_in_env(
         )
         return contract
 
+    async def invoke_setup_state(self, fn_name: str) -> None:
+        await self.invoke_test_case(fn_name)
+        assert self._hijacked_hint_locals is not None
+
+        self.tmp_state = (
```

Comment:
This seems unncessary

Reply:
Yep, `tmp_state` will be updated because it's passed by a reference.